extensions/wikia/FlowTracking/scripts/createPageTracking.js

```diff
@@ -0,0 +1,41 @@
+define(
+	'ext.wikia.flowTracking.createPageTracking',
+	['wikia.flowTracking', 'wikia.querystring', 'mw', 'wikia.document', 'wikia.window'],
+	function (flowTrack, QueryString, mw, document, window) {
+		'use strict';
+
+		var namespaceId = mw.config.get('wgNamespaceNumber'),
+			articleId = mw.config.get('wgArticleId'),
+			title = mw.config.get('wgTitle');
+
+		function trackOnEditPageLoad(editor) {
+			var qs = new QueryString(),
+				// 'flow' is the parameter passed in the url if user has started a flow already
+				flowParam = qs.getVal('flow', false),
+				tracked = qs.getVal('tracked', false);
+
+			// Do not track if the step was tracked already or article exists
+			if (tracked
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1067)